databuilder/README.md

```diff
@@ -356,6 +356,39 @@ job = DefaultJob(
 job.launch()
 ```
 
+#### [SnowflakeUsageExtractor](https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/extractor/snowflake_usage_extractor.py "SnowflakeUsageExtractor")
+An extractor that extracts table popularity metadata from a custom created Snowflake table (created by a script that may look like [this scala script](https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/example/scripts/sample_snowflake_table_usage.scala "sample_snowflake_table_usage")). You can create a DAG using the [Databricks Operator](https://github.com/apache/airflow/blob/main/airflow/providers/databricks/operators/databricks.py) and run this script within Databricks.
```

Comment:
Is there an alternative or more generic method than using this databricks example? Just thinking through how this could be used by those who don't have databricks

Reply:
You can run this anywhere you have the ability to run scala! This is databricks specific, it's just the environment we use to run scala jobs.